### PR TITLE
Fix bug of cut & paste bundles

### LINKED ISSUE #3572: Cut and Paste Bundles
Cut a bundle (a d) and then go to the last bundle and try to paste (a v), it still pastes it at the top.

### PR DESCRIPTION
**Pull request by yuqijin**
### Related issues
fixes #3572 

### Checklist

* [x] I've added a screenshot of the changes, if this is a frontend change
* [x] I've added and/or updated tests, if this is a backend change
* [x] I've run the [pre-commit.sh](https://github.com/codalab/codalab-worksheets/blob/master/pre-commit.sh) script
* [x] I've updated [docs](https://github.com/codalab/codalab-worksheets/tree/master/docs), if needed

### CODE REVIEW

File: frontend/src/components/worksheets/Worksheet/Worksheet.js [hunk 1]
```diff
@@ -519,7 +519,7 @@ class Worksheet extends React.Component {
         } else {
             // If no location for the insertion is specified,
             // insert the new item at the top of the worksheet in default.
```
Comment (epicfaace) [inline frontend/src/components/worksheets/Worksheet/Worksheet.js]:
```suggestion
            // insert the new item at the top of the worksheet by default.
```

File: frontend/src/components/worksheets/Worksheet/Worksheet.js [hunk 2]
```diff
@@ -519,7 +519,7 @@ class Worksheet extends React.Component {
...
-            actualData['after_sort_key'] = -1;
+            actualData['after_sort_key'] = 0;
```
Comment (epicfaace) [inline frontend/src/components/worksheets/Worksheet/Worksheet.js:525]: To solve issues like this in the future, maybe we should define a constant called `WORKSHEET_TOP_POS` and set it to -1, and use it in all places that depend on the worksheet top position. What do you think? (we could probably do this in another PR as this is just a general refactoring thing though)

**Review by epicfaace — APPROVED** [review]

File: frontend/src/components/worksheets/Worksheet/Worksheet.js (hunk 2 above)
Comment (yuqijin) [reply frontend/src/components/worksheets/Worksheet/Worksheet.js:525]: Yes, I think it is a good suggestion!

Comment (epicfaace) [reply frontend/src/components/worksheets/Worksheet/Worksheet.js:525]: Tracking in https://github.com/codalab/codalab-worksheets/issues/3623

**Review by epicfaace — APPROVED** [review]

**Pull request merged**